mkosi/__init__.py

```diff
@@ -2241,6 +2242,100 @@ def install_openmandriva(args: CommandLineArguments, root: str, do_run_build_scr
     disable_pam_securetty(root)
 
 
+@complete_step("Installing Gentoo")
+def install_gentoo(args: CommandLineArguments, root: str, do_run_build_script: bool) -> None:
+    try:
+        import multiprocessing
```

Comment:
I meant move to the top of the entire file where all the other imports are located. The emerge imports can stay here as they only need to be available when gentoo is built.

Reply:
done